### PR TITLE
Feat: extract users cohorts script

### PR DESCRIPTION
**Pull request by SietsmaRJ**
- Created Python script to extract cohorts and users from a fusion auth url and map the users to their corresponding cohort.
- Added pandas and numpy to Pipfile

### CODE REVIEW

**Comment by marikaris** [comment]
[M10353 As armadillo datamanager I would like to download a list of users from the fusionauth](https://trello.com/c/9mQch7NQ/1088-m10353-as-armadillo-datamanager-i-would-like-to-download-a-list-of-users-from-the-fusionauth)

File: scripts/export-users.py [hunk 1]
```diff
@@ -0,0 +1,372 @@
+#!/usr/bin/env python3
+
+"""
+User export script to aid in migration
+
+This script gets users and their assigned cohorts from Fusion Auth to Armadillo.
+It will group each user by cohort and will export it as TSV to a given export location, so that upon
+request this TSV can be used to set users to their appropriate cohorts.
```
Comment (clemens-tolboom) [inline scripts/export-users.py:8]: Isn't this "just exporting Fusion users into a TSV? Or is it a special Armadillo compatible format?

File: scripts/export-users.py [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+                        roles = np.nan
+                    else:
+                        roles = self._process_roles(registration['roles'])
+                    self.cohort_dict[identifier].add_user(current_user[self.columns_of_interest])
+                    self.cohort_dict[identifier].add_user_roles(
+                        current_user['email'].values[0],
+                        roles
+                    )
+        if user_mail:
+            self.users_not_parsed.append(user_mail)
```
Comment (clemens-tolboom) [inline scripts/export-users.py:156]: Shouldn't this be placed into `for user in users...`?

**Review by clemens-tolboom — COMMENTED** [review]
Quickscan review.

File: scripts/export-users.py (hunk 1 above)
Comment (SietsmaRJ) [reply scripts/export-users.py:8]: Simple TSV, I'll change it

**Review by marikaris — APPROVED** [review]

**Pull request merged**